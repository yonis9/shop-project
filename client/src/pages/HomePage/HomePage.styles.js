import styled from 'styled-components';

export const HomePageContainer = styled.div`
    display: flex;
    flex-direction: column;
    align-items: center;
    padding: 0 40px;

    @media only screen and (max-width: 800px) {
        padding: 0 10px ;
    }
`