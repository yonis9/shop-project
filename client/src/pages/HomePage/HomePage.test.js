import React from 'react'; 
import { shallow } from 'enzyme';

import HomePage from './HomePage';

it('should render Homepage', () => {
    expect(shallow(<HomePage />)).toMatchSnapshot();
})