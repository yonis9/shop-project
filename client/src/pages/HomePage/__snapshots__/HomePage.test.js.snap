// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Homepage 1`] = `ShallowWrapper {}`;
